src/test/debuginfo/repr-u128.rs

```diff
@@ -0,0 +1,17 @@
+// compile-flags: --emit=llvm-ir -C debuginfo=2
+#![feature(repr128)]
+
+#[repr(u128)]
+pub enum Foo {
+    Lo,
+    Hi = 1 << 64,
+}
+
+pub fn foo() -> Option<Foo> {
```

Comment:
You don't need this in the debuginfo test (which we should rename to "debugger" tests, really. cc @michaelwoerister)

Reply:
@eddyb I removed it.